packages/vm/lib/evm/eei.ts

```diff
@@ -583,6 +583,14 @@ export default class EEI {
     return this._state.accountIsEmpty(address)
   }
 
+  /**
+   * Returns true if account exists in the state trie (it can be empty). Returns false if the account is `null`.
+   * @param address - Address of account
+   */
+  async accountExists(address: Buffer): Promise<boolean> {
+    return this._state.accountExists(address)
```

Comment:
This is currently not covered by tests:
https://codecov.io/gh/ethereumjs/ethereumjs-vm/compare/2009f18c81f2fe253e846333ace57eb77625dd60...681418a97f5e27b56da0f5ca63a6fd4acb244c2b/src/packages/vm/lib/evm/eei.ts?before=packages/vm/lib/evm/eei.ts#L591

Reply:
Added a test for this (`StateManager` accountExists is actually being tested; the `accountExists` of EEI is actually a proxy function but we should still test that)